resources/external_jointcal.yaml

```diff
@@ -133,7 +133,7 @@ data:
   run: HSC/external
   records:
   - dataset_id:
-    - 10000
+    - 20000
```

Comment:
Are these okay as integers rather than UUID?

Reply:
I believe the code converts them to UUIDs with `UUID(int=...)`. At least I saw these UUIDs when I printed them.